grass7/raster/r.null.all/r.null.all.html

```diff
@@ -0,0 +1,57 @@
+<h2>DESCRIPTION</h2>
+
+<em>r.null.all</em> manages NULL (no-data) values in all raster maps
+in the current mapset. Selection can be modified using <b>pattern</b>
+and <b>exclude</b> options. The option <b>matching</b> specifies
+the type of search pattern use. Python users will find the extended
+regular expression syntax (marked as <em>extended</em>) as most
+familiar, while Bash users may want to use <em>wildcards</em>
+(glob patterns).
+
+<h2>EXAMPLE</h2>
+
+All the following examples are using the <b>-d</b> flag to run in a
+<em>dry run</em> mode so that no maps are actually modified.
+
+Set all values 1 to NULL in all raster maps in the current mapset:
+
+<div class="code"><pre>
+r.null.all setnull=1 -d
+</pre></div>
+
+Change all NULL to zero in all raster maps in the current mapset
+which begin with letter t and their name contains at least one other
+character (using the extended regular expressions):
+
+<div class="code"><pre>
+!r.null.all null=0 pattern="^t.+" m=extended -d
```

Comment:
```suggestion
r.null.all null=0 pattern="^t.+" matching=extended -d
```
(please avoid abbreviations in manual page examples)